### ISSUE TITLE
Response interpolation needs improvement

### ISSUE THREAD

**Issue by vetlewi** [oslocyclotronlab/ompy#64]
I tried to use the response interpolation for NaI (Magne have an early set of files for this ready) and it doesn't seem to work very well.

This is the response matrix that are produced by choosing MaMa opt. 3:
![mama_default_interpolation](https://user-images.githubusercontent.com/5293376/66830916-d9b11d80-ef56-11e9-9899-1884bacf5f8e.png)
As one can see, the result is a smooth response matrix as one expects.

Using the folder option (opt. 13), we get the following response matrix:
![mama_interpolation_folder](https://user-images.githubusercontent.com/5293376/66831020-154be780-ef57-11e9-86a0-1867b37b22cb.png)
This is obviously not very good, but the matrix is smooth and probably would work reasonably well.

Lastly I tried to perform the interpolation with OMpy using the `om.interpolate_response()` method. This gave this matrix:
![ompy_interpolation](https://user-images.githubusercontent.com/5293376/66831270-88edf480-ef57-11e9-93f5-1c23d71725a0.png)
Which is just horrible! The horizontal lines corresponds to the experimental input energies.

I belive that part of the reason this fails is that a linear interpolation is used for the relative intensities of the FE, SE, DE, 511 and Compton:
https://github.com/oslocyclotronlab/ompy/blob/06a81457152c1afd6b80ee2e92903f9fea28306c/ompy/response.py#L201-L207
And one should probably probably use a more physical model of the intensities for the peaks.

There might also be a bug in the implementation of the "fan-interpolation" of the Compton spectra.

**Comment by fzeiser** [comment]
This is indeed strange! Both, that mama option 3 and 13 are not the same, and, of course, the interpolation with ompy is just nuts.

If you want to have a workaround to get starting on whatever you wanted, I'd suggest you to 
- read the spectra you work on into mama
- create the response matrix (after reading the spectrum it should have the binning you need)
- use `gr` to get the response matrix and write it to disk
- read it in ompy

Note: We're still also working on #39, the quality of the unfolding. It's hard to imitate what mama does, as it definitively is not just what was written in the paper...

**Comment by vetlewi** [comment]
> This is indeed strange! Both, that mama option 3 and 13 are not the same, and, of course, the interpolation with ompy is just nuts.
> 
> If you want to have a workaround to get starting on whatever you wanted, I'd suggest you to
> 
> * read the spectra you work on into mama
> * create the response matrix (after reading the spectrum it should have the binning you need)
> * use `gr` to get the response matrix and write it to disk
> * read it in ompy
> 
> Note: We're still also working on #39, the quality of the unfolding. It's hard to imitate what mama does, as it definitively is not just what was written in the paper...

I will use this solution for now. As for the interpolation it might be good to do a look in the literature for what shapes is expected for the FE, SE, DE, Compton & 511 efficiencies. Maybe we get a better interpolation if we use a 'physical' model?

**Comment by fzeiser** [comment]
We could do so, yes. But we should also think about how important which of these cases is. The most common way now is to use geant4 simulation. There, we just specify a fine enough grid to make any error due to the interpolation irrelevant.

**Comment by vetlewi** [comment]
> We could do so, yes. But we should also think about how important which of these cases is. The most common way now is to use geant4 simulation. There, we just specify a fine enough grid to make any error due to the interpolation irrelevant.

I agree, for a fine enough grid the interpolation error will be negligible. However, the interpolation (at least for the FE) should probably be logarithmic, rather than linear...

**Comment by fzeiser** [comment]
It seems like something went wrong in the interpolation of the Compton background. It's a function that was still written by Jørgen, and I never went  through it thoroughly. I'll try to find the error... It could also e some assumption about the input files before interpolation.

**Comment by fzeiser** [comment]
Further treated in #65, as I was missing `push -f` rights
